### PR TITLE
main - feature merge @3ecc004e2

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-1: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

Maintenance/feature PR pulling in `master` at 3ecc004e2

Merged by branching off master, then pulling in the feature branch.  

Most of the merge went in cleanly, however reviewers should pay close attention to 
fb9ff54  as a few manual revisions to the includes were required, etc. 

Most notably includes #2308 :tada:

### CODE REVIEW

File: example/cumulus-tf/main.tf [hunk 1]
```diff
@@ -156,7 +156,8 @@ module "cumulus" {
     "mboyd",
     "menno.vandiermen",
     "mobrien84",
-    "npauzenga"
+    "npauzenga",
+    "ds_jennifertran"
```
Comment (Jkovarik) [inline example/cumulus-tf/main.tf:160]: 🎉

File: example/deployments/cumulus/jl-tf.tfvars [hunk 2]
```diff
@@ -40,3 +40,6 @@ key_name      = "jl"
 
 include_orca = true
 drop_database = "False"
+
+distribution_url = "https://jwhwz6qg4j.execute-api.us-east-1.amazonaws.com:7000/DEV/"
```
Comment (Jkovarik) [inline example/deployments/cumulus/jl-tf.tfvars:44]: 🤔

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,32 +8,88 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### BREAKING CHANGES
 
-- `@cumulus/api-client/granules.getGranule` now returns the granule record from the GET `/granules/<granuleId>` endpoint, not the raw endpoint response
```
Comment (markdboyd) [inline CHANGELOG.md:11]: I think this note got lost in the merge?

**Review by markdboyd — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (Jkovarik) [reply CHANGELOG.md:11]: Whoops!    Will sort.

**Pull request merged**